zkevm-circuits/src/rlp_circuit_fsm/test.rs

```diff
@@ -109,26 +109,47 @@ fn test_l1_msg_tx() {
+    ];
+
+    let mut txs: Vec<Transaction> = vec![];
+    for (idx, bytes) in test_bytes.into_iter().enumerate() {
+        let raw_tx_rlp_bytes = hex::decode(bytes).expect("decode tx's hex shall not fail");
+
+        let eth_tx = EthTransaction::decode(&Rlp::new(&raw_tx_rlp_bytes))
+            .expect("decode tx's rlp bytes shall not fail");
+
+        let eth_tx_req: Eip1559TransactionRequest = (&eth_tx).into();
+        let typed_tx: TypedTransaction = eth_tx_req.into();
+        let rlp_unsigned = typed_tx.rlp().to_vec();
+
+        let tx = Transaction::new_from_rlp_bytes(
+            idx + 1,
+            TxType::Eip1559,
+            raw_tx_rlp_bytes,
+            rlp_unsigned,
+        );
+
+        txs.push(tx);
+    }
```

Comment:
```suggestion
    let txs = test_bytes.into_iter().enumerate().map(|(idx, bytes)| {
        let raw_tx_rlp_bytes = hex::decode(bytes).expect("decode tx's hex shall not fail");

        let eth_tx = EthTransaction::decode(&Rlp::new(&raw_tx_rlp_bytes))
            .expect("decode tx's rlp bytes shall not fail");

        let eth_tx_req: Eip1559TransactionRequest = (&eth_tx).into();
        let typed_tx: TypedTransaction = eth_tx_req.into();
        let rlp_unsigned = typed_tx.rlp().to_vec();

        Transaction::new_from_rlp_bytes(
            idx + 1,
            TxType::Eip1559,
            raw_tx_rlp_bytes,
            rlp_unsigned,
        )
    })
    .collect::<Vec<Transaction>>();
```

Just a nitpick, for code quality. Might need formatting.

Reply:
Style adopted.